annex/chapter1/adv_examples/include/distance_calculation.h

```diff
@@ -0,0 +1,26 @@
+#include <klepsydra/core/subscriber.h>
```

Comment:
Add header include guard on the first line of the file similar to other header files.

Reply:
Done